### DISCUSSION TITLE
Tutorials

### DISCUSSION THREAD

**Discussion by antonkomarev** [cybercog/laravel-love#210] [Show and tell]
| Version | Link | Language | Author |
| :---- | :---- | :---- | :---- |
| 5.x | [Likes y Dislikes con el paquete Laravel Love](https://styde.net/likes-y-dislikes-con-el-paquete-laravel-love/) | 🇪🇸 ES | [Carlos Fernandes](https://github.com/carlosjfernandes) |

*Feel free to post your links in comments. They will be added to the first post.*

**Comment by MoatazAbdAlmageed** [comment]
@antonkomarev Please mention that this post not updated by the latest package  updates

**Comment by antonkomarev** [comment]
@MoatazAbdAlmageed that's why I've added `Version` column to the table with `5.x` value and now we have `7.x` in master branch.

**Comment by vesper8** [comment]
Coming from v5, v8 is overwhelmingly more complex. I'm going to stick with it and I find it interesting everything that you can do now. I wish there was a simple tutorial that explained how to do very simple things like like/dislike with v8. Basically, for people who were using v5 and only needed to do simple liking/disliking, and decided to upgrade to v8 and beyond for the sake of staying up-to-date.

I can't find an upgrade guide either but maybe I'm just not looking in the right place

**Comment by antonkomarev** [comment]
All upgrade guides are in one place: https://github.com/cybercog/laravel-love/blob/master/UPGRADING.md

**Comment by vesper8** [comment]
oops.. I missed that! You should add a link to this from the documentation (if there isn't one already)

Thanks! That is very helpful

**Comment by antonkomarev** [comment]
Nice catch! I've added Upgrade Guide to the documentation: https://laravel-love.readme.io/v8.0/docs/upgrade-guide and to README.md file because it was missing after migrating documentation to external doc service.

**Comment by iamB0rgy** [comment]
Why there is no complete tutorial on the web?

**Reply by antonkomarev** [reply]
Because we are waiting for the Hero ))

**Reply by iamB0rgy** [reply]
Hope this helps! [https://github.com/cybercog/laravel-love/discussions/74#discussioncomment-4663418](url)

**Reply by antonkomarev** [reply]
Thanks! 🙏
